packages/visualizations/addon/components/navi-visualizations/table.js

```diff
@@ -15,7 +15,8 @@ import layout from '../../templates/components/navi-visualizations/table';
 import { formatItemDimension } from '../../helpers/mixed-height-layout';
 import groupBy from 'lodash/groupBy';
 import { canonicalizeMetric } from 'navi-data/utils/metric';
-import { computed, get, set, setProperties } from '@ember/object';
+import { computed, get, set, setProperties } from "@ember/object";
```

Comment:
Why did we switch to double quotes here?

Reply:
I think it is caused by my code editor plugin